client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/PlacesAdditionalForms/SchoolEventForm.tsx

```diff
@@ -37,15 +37,15 @@ const SchoolEventForm : React.FC = () : JSX.Element => {
 
     const formClasses = useFormStyles();
     const { setInteractionEventDialogData, interactionEventDialogData } = useContext(InteractionEventDialogContext);
-    const { placeSubType } = interactionEventDialogData;
+    const { placeSubType, grade } = interactionEventDialogData;
 
     const [grades, setGrades] = useState<string[]>([]);
 
     React.useEffect(() => {
         let gradesOptions : string[] = [];
         if (placeSubType === elementarySchool) gradesOptions = elementarySchoolGrades;
         else if (placeSubType === highSchool) gradesOptions = highSchoolGrades;
-        if (placeSubType > 0) setInteractionEventDialogData({...interactionEventDialogData, grade: gradesOptions[0]});
+        if (placeSubType !== -1 && !grade) setInteractionEventDialogData({...interactionEventDialogData, grade: gradesOptions[0]});
```

Comment:
I think !grade is enough, since if !grade is true then placeSubType won't be -1

Reply:
What happens is the grade isn’t defined when the placeSubType is as well, so it updates the grade and the placeSubType as -1 with it. I added the condition in order to prevent it